### PR TITLE
`list-refundables`: show refundable amount, not swap amount

### PR DESCRIPTION
**Pull request by ok300**
The refundable amount is based on the locked up utxo(s), not the initial swap amount.

To retrieve this, we have to make a chain service lookup.

### CODE REVIEW

**Review by hydra-yse — APPROVED** [review]
utACK

File: lib/core/src/sdk.rs [hunk 1]
```diff
@@ -1740,12 +1740,27 @@ impl LiquidSdk {
...
+        let chain_swaps = self.persister.list_refundable_chain_swaps()?;
+
+        let mut refundables = vec![];
+        for chain_swap in chain_swaps {
+            let script_pubkey = chain_swap.get_lockup_swap_script_pubkey(self.config.network)?;
+            let script_balance = self
+                .bitcoin_chain_service
+                .lock()
+                .await
+                .script_get_balance(script_pubkey.as_script())?;
```
Comment (roeierez) [inline lib/core/src/sdk.rs:1752]: Electrum has a batched version of this API: https://docs.rs/electrum-client/latest/electrum_client/trait.ElectrumApi.html#tymethod.batch_script_get_balance
Perhaps we can use get all balances in one call.

**Review by roeierez — CHANGES_REQUESTED** [review]

File: lib/core/src/sdk.rs (hunk 1 above)
Comment (ok300) [reply lib/core/src/sdk.rs]: Done.

File: lib/core/src/model.rs [hunk 2]
```diff
@@ -806,17 +828,9 @@ impl ReceiveSwap {
 pub struct RefundableSwap {
     pub swap_address: String,
     pub timestamp: u32,
+    /// Amount that is refundable, which is different than the swap amount
```
Comment (dangeross) [inline lib/core/src/model.rs:831]: It *might* be the same right? So maybe its clearer to write
```suggestion
    /// Amount that is refundable, from all UTXOs
```

**Review by dangeross — COMMENTED** [review]
Looks good, just a few comments

**Review by roeierez — APPROVED** [review]
LGTM

File: lib/core/src/model.rs [hunk 3]
```diff
@@ -608,6 +609,27 @@ impl ChainSwap {
...
 
+    /// Returns the lockup script pubkey for Receive Chain Swaps
+    pub(crate) fn get_receive_lockup_swap_script_pubkey(
+        &self,
+        network: LiquidNetwork,
+    ) -> SdkResult<ScriptBuf> {
+        let swap_script = self.get_lockup_swap_script()?.as_bitcoin_script()?;
+        let script_pubkey = swap_script
+            .to_address(network.as_bitcoin_chain())
+            .map_err(|e| anyhow!("Error getting script address: {e:?}"))?
```
[the remark this answers is not in the document]
Comment (ok300) [reply lib/core/src/model.rs]: Done: 64e282b7f89d2a982bdd659888855bdda343dbb1

File: lib/core/src/model.rs (hunk 2 above)
Comment (ok300) [reply lib/core/src/model.rs]: Good point, done: 723679ea0a2bd6acec6be71b420c9e1313c25ee5

**Review by dangeross — APPROVED** [review]

**Pull request merged**